frontend/src/chooseAdventure/introView.js

```diff
@@ -18,7 +18,11 @@ class IntroView extends React.Component {
                     <div className={'col-6'}>
                         {this.props.desc}
                     </div>
-                    <img className= 'col-6' src={this.props.imgFile} alt="Sample" />
+                    <img
+                        className= 'col-6' src={this.props.imgFile}
+                        alt='intro_image'
```

Comment:
Please add descriptive alt text here -- it should describe the content and function of the image. See https://www.w3.org/WAI/tutorials/images/ for much more, but in a nutshell: alt text is super important for users who are using assistive technology like screen readers in order to interact with our site.